packages/runtime/test/consumption/iqlQuery.test.ts

```diff
@@ -110,6 +110,7 @@ describe("IQL Query", () => {
         expect(sLocalRequest.status).toBe(LocalRequestStatus.Draft);
         expect(sLocalRequest.content.items).toHaveLength(1);
         expect(sLocalRequest.content.items[0]["@type"]).toBe("ReadAttributeRequestItem");
+        if (sLocalRequest.content.items[0]["@type"] !== "ReadAttributeRequestItem") throw new Error("Unexpected type.");
```

Comment:
I don't understand this line. Right above you check the type.

Reply:
If you remove this line, you will get an error. This is because of TypeScript's inability to refine types based on assertions like `expect`.